feat(nacls): add doc for NACL public beta (#4705)
scaleway/docs-content: pages/vpc/faq.mdx

Managed Databases do not currently support VPC routing - see our [dedicated documentation](/vpc/reference-content/understanding-routing/#limitations)

### Can I control traffic flow between my VPC's Private Networks?

> Yes, use the [Network ACL feature](/vpc/how-to/manage-nacl) to filter packets flowing between the different Private Networks of your VPC. By default all traffic is allowed to pass, until you start to add rules to the VPC's NACL.

Reviewer: ```suggestion
Yes, use the [Network ACL feature](/vpc/how-to/manage-nacl) to filter packets flowing between the different Private Networks of your VPC. By default, all traffic is allowed to pass, until you start to add rules to the VPC's NACL.
```

> Yes, use the [Network ACL feature](/vpc/how-to/manage-nacl) to filter packets flowing between the different Private Networks of your VPC. By default, all traffic is allowed to pass, until you start to add rules to the VPC's NACL.

Author: ```suggestion
Yes, use the [Network ACL feature](/vpc/reference-content/understanding-nacls/) to filter packets flowing between the different Private Networks of your VPC. By default, all traffic is allowed to pass, until you start to add rules to the VPC's NACL.
```

<Message type="important">
NACLs are currently in Public Beta, and configurable only via the [VPC API](https://www.scaleway.com/en/developers/api/vpc/) and developer tools. This feature will be coming soon to the Scaleway console.
</Message>

> #"# How are NACLs different to security groups?

Reviewer: ```suggestion
## How are NACLs different from security groups?
```

Author: I want to fight for this one, but googling tells me it's more British than American ;-)
